backend/src/main/java/io/papermc/hangar/db/dao/v1/ProjectsApiDAO.java

```diff
@@ -137,18 +137,18 @@ SELECT count(hp.id)
         "       JOIN user_project_roles upr ON p.id = upr.project_id " +
         "       JOIN users u ON upr.user_id = u.id " +
         "       JOIN roles r ON upr.role_type = r.name " +
-        "   WHERE lower(p.slug) = lower(:slug) " +
+        "   WHERE p.id = :id " +
```

Comment:
we only ever use the id of the project, so we don't need to select from projects, we can select from user_project_roles instead, saving one join